packages/core/src/RenderingEngine/helpers/createVolumeActor.ts

```diff
@@ -61,7 +62,7 @@ async function createVolumeActor(
   // types of volumes which might not be composed of imageIds would be e.g., nrrd, nifti
   // format volumes
   if (imageVolume.imageIds) {
-    await setDefaultVolumeVOI(volumeActor, imageVolume);
+    await setDefaultVolumeVOI(volumeActor, imageVolume, use16BitTexture);
```

Comment:
This option should go in an options object, as there may well be others in the future.